app/views/posts/show.html.erb

```diff
@@ -19,5 +24,12 @@
         </div>
       <% end %>
     </div>
+    <div>
+      <%= form_with(model: Comment.new, url: user_post_comments_path(@user, @post), method: :post) do |form| %>
+        <%= form.label :text, 'Comment' %>
+        <%= form.text_area :text %>
+        <%= form.submit 'Add Comment' %>
+      <% end %>
+    </div>
```

Comment:
Nice job creating the form for the comments. :1st_place_medal:  However, in line with the project requirements and ruby best practices, kinldy consider creating this form in a `new.html.erb` file under views/comments (`views/comments/new.html.erb`) :pray: